### PR TITLE
Advanced threat detection can be enabled when a license is created.

### PR DESCRIPTION
**Pull request by mrudatsprint**
refers to [https://github.com/FusionAuth/fusionauth-site/issues/2584](https://github.com/FusionAuth/fusionauth-site/issues/2584)

### screenshot
atd-enabled

### CODE REVIEW

File: astro/src/content/docs/operate/secure-and-monitor/advanced-threat-detection.mdx [hunk 1]
```diff
@@ -23,9 +23,7 @@ Flagging and responding to suspicious behavior is a part of any cybersecurity pr
 Advanced Threat Detection, a feature that brings best-practice functionality to help you deal with bizarre, possibly malicious behavior around logins, registrations, user creation, and user updates.
 
 <Aside type="note">
-Turning this feature on requires a support ticket, as it is being rolled out on an account by account basis.
-
-To enable Advanced Threat Detection, submit a [support ticket through your account portal](https://account.fusionauth.io/account/support) and request it be turned on.
+Advanced Threat Detection can be enabled when the user creates a license.
```
Comment (mooreds) [inline astro/src/content/docs/operate/secure-and-monitor/advanced-threat-detection.mdx:26]: Can it be enabled at any other time? Like can I update my existing license to enable this?

If not, what should I do if I need one?

**Review by mooreds — CHANGES_REQUESTED** [review]
One question where I think things could be clearer for our end user.

File: astro/src/content/docs/operate/secure-and-monitor/advanced-threat-detection.mdx (hunk 1 above)
Comment (mooreds) [reply astro/src/content/docs/operate/secure-and-monitor/advanced-threat-detection.mdx:26]: It's your PR :) . Please make whatever changes you deem appropriate.

Comment (mrudatsprint) [reply astro/src/content/docs/operate/secure-and-monitor/advanced-threat-detection.mdx]: I have updated when ATD can be enabled.

**Comment by mooreds** [comment]
Is this intentionally still a draft?

**Comment by mrudatsprint** [comment]
> Is this intentionally still a draft?

I marked the PR ready for review.

**Review by mooreds — APPROVED** [review]

**Pull request merged**